crates/levm/src/vm.rs

```diff
@@ -137,3 +171,187 @@ impl Memory {
         self.data[dest_offset..dest_offset + size].copy_from_slice(&temp);
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Let's move these tests to `crates/levm/tests/vm_tests.rs`.

Reply:
Will do!